packages/gestalt/src/index.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import 'babel-polyfill';
```

Comment:
@chrislloyd @bishwei This is probably the most controversial change, wdyt? Any suggestions of an alternative?

Reply:
Yeah, I'm a hard 👎  on this - we should either update `browsers` in `package.json` to ship with IE11 support or we should figure out something else. This brings all of the regenerator runtime into Gestalt.

Reply:
imo, this should be on the consumer, not on gestalt.

also, this would definitely bloat our js size a lot